Swiften/Client/Client.cpp

```diff
@@ -74,6 +75,9 @@ Client::Client(const JID& jid, const SafeString& password, NetworkFactories* net
     jingleSessionManager = new JingleSessionManager(getIQRouter());
     blockListManager = new ClientBlockListManager(getIQRouter());
 
+    mixSupport = false;
```

Comment:
I just used this as a bool to pass to requestRoster. However it is still redundant.